index.js

```diff
@@ -104,13 +104,15 @@ module.exports.resultFormatter = exports.defaultFormatter
  * @param {string[]} [filterPaths] A list of directories to allow linting of, or [] for all.
  * @param {Object|string|null} [ruleset] A custom ruleset object with the same structure as the JSON ruleset configs, or a string path to a JSON config.
  * Set to null for repolinter to automatically find it in the repository.
+ * @param {boolean} [encodedIsUsed] If true, repolinter expects base64 encoded string in the ruleset, we will decode the base64 encoded ruleset.
  * @param {boolean} [dryRun] If true, repolinter will report suggested fixes, but will make no disk modifications.
  * @returns {Promise<LintResult>} An object representing the output of the linter
  */
 async function lint(
   targetDir,
   filterPaths = [],
   ruleset = null,
+  encodedIsUsed = false,
```

Comment:
do we really need the flag?  What if instead we simply detected whether it's encoded or not?